index.js

```diff
@@ -1,10 +1,15 @@
 'use strict';
 
+require('es6-promise').polyfill();
```

Comment:
Should we really patch `window` in a client lib? I hated when libs were doing that.